codeforboston/cliff-effects
# Fixes #545, Form routes

Test plan:

- Mark one or more of the benefits in the first step.
- Click on steps in the step bar. The URL should change and you should not see any dialog asking you to confirm navigation.
- Click on a link outside the visit workflow (e.g. Home, About)
- You should see a navigation confirmation dialog

- Reload the page while on one of the later steps (not the first step)
- Before the page changes, you should see a navigation confirmation dialog
- After reloading, the visit should go back to the first step

- If you have not selected any benefits, you should not see any confirmation dialog, even when going outside the visit workflow.
